main.py

```diff
@@ -46,15 +43,15 @@
     # ----------------------------------------
     #               MARKET
     # ----------------------------------------
-    elif message.content.startswith('!m'):
+    if message.content.startswith('!m'):
 
         # When a specific emoji has been chosen.
         args = message.content.split(' ')
         if len(args) > 1: #shoudnt this be 2? or does len actually return count?
 
             emoji = args[1]
             amountS = args[2]
-            if (!amountS)
+            if !amountS
```

Comment:
This requires a colon at the end

Reply:
It should be `not` instead of `!` as well